qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -10,15 +10,18 @@
 
 log = logging.getLogger(__name__)
 
+
 def extract_schedule_and_retention_spec(spec=[]):
     schedule = set([s[0] for s in spec])
     retention = set([s[1] for s in spec])
     return (schedule, retention)
 
+
 def seconds_upto_next_schedule(time_from, timo):
     ts = int(time_from)
     return ((int(ts / 60) * 60) + timo) - ts
 
+
```

Comment:
unrelated change.

Reply:
unrelated changes are mostly to keep `flake8` happy

Reply:
separate pr please.

Reply:
ack
restored `flake8` changes as much as possible
however, I didn't restore the spurious spaces that I deleted